packages/lib/server/repository/user.ts

```diff
@@ -73,6 +74,28 @@ const userSelect = Prisma.validator<Prisma.UserSelect>()({
   isPlatformManaged: true,
 });
 
+type UserWithNotToBeUsedSelectedCalendars<TCalendarProps, TUserProps> = TUserProps & {
+  selectedCalendars: TCalendarProps[];
+};
+
+export function enrichWithSelectedCalendars<
```

Comment:
It ensures that `selectedCalendars` can't be used because now it holds both all eventTypes' selectedCalendars plus the user calendars. 

Existing could would expect selectedCalendars to be just User Calendars, so we removed it and added more clear `userLevelSelectedCalendars` which is now the alternative of `user.selectedCalendars`